contrib/ansible/roles/objectstorage/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Set facts
+  set_fact:
+    minio_path: "/opt/minio"
+    minio_client_path: "/opt/mc"
+    minio_data_path: "/opt/minio-data"
+    minio_endpoint: "http://127.0.0.1:9000"
+    minio_cluster_name: "local"
+    minio_access_key: "user"
+    minio_secret_key: "password"
+    bucket_name: "bucket"
+
+- name: Download minio
+  get_url:
+    url: https://dl.minio.io/server/minio/release/linux-amd64/minio
+    dest: "{{ minio_path }}"
+    mode: 0775
+
+- name: Download minio client
+  get_url:
+    url: https://dl.minio.io/client/mc/release/linux-amd64/mc
+    dest: "{{ minio_client_path }}"
+    mode: 0775
+
+- name: Create minio data directory
+  file:
+    path: "{{ minio_data_path }}"
+    state: directory
+    mode: 0755
+
+- name: Run minio server
+  shell: "{{ minio_path }} server {{ minio_data_path }} &"
```

Comment:
You should rather setup a service - one option is to use scripts found here - https://github.com/minio/minio-service